### PR TITLE
Release/8.0.0

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,5 @@
-version=7.1.1
-iexecCommonVersion=6.0.0
+version=8.0.0-rc0
```
Comment (jeremyjams) [inline gradle.properties:1]:
```suggestion
version=8.0.0
```

File: gradle.properties [hunk 2]
```diff
@@ -1,2 +1,5 @@
-version=7.1.1
-iexecCommonVersion=6.0.0
+version=8.0.0-rc1
```
Comment (jeremyjams) [inline gradle.properties:1]:
```suggestion
version=8.0.0
```

**Review by jbern0rd — APPROVED** [review]

File: gramine/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## How to build your Gramine application
+
+```shell
+docker build -t gramine-tee-worker-post-compute:latest -f gramine/Dockerfile  .
```
Comment (jbern0rd) [inline gramine/README.md:4]: Do we change the image name to be closer to the latest convention ?

Comment (jeremyjams) [reply gramine/README.md]: Yes, is https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/74/commits/dfb01e7f6e4ae4e8a395983938ded111b0b26ae1 fine for you?

Comment (jbern0rd) [reply gramine/README.md]: Thanks !

**Pull request merged**